From 1f5c2666d7d7153b5f6b0988caadd0120ba3b824 Mon Sep 17 00:00:00 2001
From: Mark Hatle <mhatle@xilinx.com>
Date: Mon, 6 Jun 2022 12:29:07 -0700
Subject: [PATCH] CMakeLists.txt: Remove REGEX

We want to use the Yocto Project settings, including -march and -mtune.

This fixes:
| aarch64-xilinx-linux-g++: warning: 8-a+crc: linker input file unused because linking not done
| aarch64-xilinx-linux-g++: error: 8-a+crc: linker input file not found: No such file or directory

Since the regex doesn't look for numbers, - or +.

Signed-off-by: Mark Hatle <mhatle@xilinx.com>
---
 prj/cmake/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/prj/cmake/CMakeLists.txt b/prj/cmake/CMakeLists.txt
index dbe492f0..2a120605 100644
--- a/prj/cmake/CMakeLists.txt
+++ b/prj/cmake/CMakeLists.txt
@@ -52,8 +52,6 @@ else()
 	add_definitions(-DSIMD_VERSION="unknown")
 endif()
 
-string(REGEX REPLACE "(-march=[a-z]*)|(-mtune=[a-z]*)|(-msse[0-9,\\.]*)|(-mavx[0-9]*)|(-mfma)" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
-
 if(CMAKE_BUILD_TYPE STREQUAL "Release")
     set(COMMON_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -std=c++11 -O3")
 elseif(CMAKE_BUILD_TYPE STREQUAL "Debug")
-- 
2.17.1

